README.md

```diff
@@ -0,0 +1,141 @@
+<div align="center">
+    <img src="img.png" width="20%" height="20%" alt="OneVoiceEcho-logo">
```

Comment:
I put that image reference as a placeholder. I think @dave-cao we can add the logo to the assets folder, right? I copied the image from Figma. As an alternative, we could use an SVG; here's the markup(it looks long, hence why I put the image for simplicity) for that svg:

```
<svg width="114" height="114" viewBox="0 0 114 114" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
        <rect width="114" height="114" fill="url(#pattern0_2255_4503)"/>
        <defs>
        <pattern id="pattern0_2255_4503" patternContentUnits="objectBoundingBox" width="1" height="1">
        <use xlink:href="#image0_2255_4503" transform="matrix(0.00328273 0 0 0.00335708 -0.171819 -0.113821)"/>
        </pattern>
        <image id="image0_2255_4503" width="2613" height="356" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACjUAAAFkCAYAAABi7CsSAAAACXBIWXMAABYlAAAWJQFJUiTwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAH9CSURBVHgB7N1bmiTFme7710v0fpZ6g0SNANMIVIxAzgQWyQgILlqL3TfACMoZAcXNWiX6opwRkPQE0jUClUaQphFQOnSrl4CKnR9e3hGZlYfIMHN3O/x//XhngaCACHczc7PXPpMAAAAAAAAAAAAAAAAAAAAS0AgAAAAAAAAAAAAAAAAAgHU4hfECAAAAAAAAAAAAAAAAAACIYBtwnQvFeSAAAAAAAAAAAAAAAAAAAIAEEGoEAAAAAAAAAAAAAAAAAABJINQIAAAAAAAAAAAAAAAAAACSQKgRAAAAAAAAAAAAAAAAAAAkgVAjAAAAAAAAAAAAAAAAAABIAqFGAAAAAAAAAAAAAAAAAACQBEKNAAAAAAAAAAAAAAAAAAAgCYQaAQAAAAAAAAAAAAAAAABAEgg1AgAAAAAAAAAAAAAAAACAJBBqBAAAAAAAAAAAAAAAAAAASSDUCAAAAAAAAAAAAAAAAAAAkkCoEQAAAAAAAAAAAAAAAAAAJIFQIwAAAAAAAAAAAAAAAAAASAKhRgAAAAAAAAAAAAAAAAAAkARCjQAAAAAAAAAAAAAAAAAAIAmEGgEAAAAAAAAAAAAAAAAAQBIINQIAAAAAAAAAAAAAAAAAgCQQagQAAAAAAAAAAAAAAAAAAEkg1AgAAAAAAAAAAAAAAAAAAJJAqBEAAAAAAAAAAAAAAAAAACSBUCMAAAAAAAAAAAAAAAAAAEgCoUYAAAAAAAAAAAAAAAAAAJAEQo0AAAAAAAAAAAAAAAAAACAJhBoBAAAAAAAAAAAAAAAAAEASCDUCAAAAAAAAAAAAAAAAAIAkEGoEAAAAAAAAAAAAAAAAAABJINQIAAAAAAAAAAAAAAAAAACSQKgRAAAAAAAAAAAAAAAAAAAkgVAjAAAAAAAAAAAAAAAAAABIAqFGAAAAAAAAAAAAAAAAAACQBEKNAAAAAAAAAAAAAAAAAAAgCYQaAQAAAAAAAAAAAAAAAABAEgg1AgAAAAAAAAAAAAAAAACAJBBqBAAAAAAAAAAAAAAAAAAASSDUCAAAAAAAAAAAAAAAAAAAkkCoEQAAAAAAAAAAAAAAAAAAJIFQIwAAAAAAAAAAAAAAAAAASAKhRgAAAAAAAAAAAAAAAAAAkARCjQAAAAAAAAAAAAAAAAAAIAmEGgEAAAAAAAAAAAAAAAAAQBIINQIAAAAAAAAAAAAAAAAAgCQQagQAAAAAAAAAAAAAAAAAAEkg1AgAAAAAAAAAAAAAAAAAAJJAqBEAAAAAAAAAAAAAAAAAACSBUCMAAAAAAAAAAAAAAAAAAEgCoUYAAAAAAAAAAAAAAAAAAJAEQo0AAAAAAAAAAAAAAAAAACAJhBoBAAAAAAAAAAAAAAAAAEAS3hBQmb+ffOx+kNxWPzr740aNu7jesV9vt3r74sfFtX17+9PP2zXSi4v//+K//7iR10u92D7Y/nmrrbf/rdGDFxcPmv/56VMvAAAAAAAAAAAAAAAAAMCNGgGF2Z58+vZ/6B/upb5/NAUWt9vtIwspXtzwTiu6+HfwF/8+3sKPW23/dPHHzx/oDf/m6f9+LgAAAAAAAAAAAAAAAKA+Wx3PX1y/EopCqBFZswDjX/Wf7VYv3c/0s1+/3G7btYOLAZ5fPJL+ZfPj7x/ojedv6n88b06fvBAAAAAAAAAAAAAAAABQLkKNuIRQI7Lyt5N/ffSj/tEWEGA81PNGzfOXzcvfP9A/PaeiIwAAAAAAAAAAAAAAAApDqBGXEGpE0v5+8rH7h344ebD92W+kl+3FLfu2KnbxwPqtmufb5uW3/6SfDT8/feoFAAAAAAAAAAAAAAAA5ItQIy4h1Iik2HHSf9N/PWpebt9/2WxPKqjEGOp5s22G7YPm27dOnw4CAAAAAAAAAAAAAAAA8kKoEZcQasTqLMj4V/3HSbN98D7VGI/X/NRIN8PLZvvtL06/OhUAAAAAAAAAAAAAAACQPkKNuIRQI1YxVWTUdvtYevmIIGNs2xeNHpxum+ZrKjgCAAAAAAAAAAAAAAAgYYQacQmhRizqrycft3a09LZ5uSHIuIypguPPmubzn58+9QIAAAAAAAAAAAAAAADSQagRlxBqxOx+qsr48j8/ubjbTi7+8JGwpufbZvvlW/p/T5vTJy8EAAAAAAAAAAAAAAAArItQIy4h1IjZ/FSVcbv9cKuXJ1RlTAvVGwEAAAAAAAAAAAAAAJAIQo24hFAjorMwo7bbxxftTStkoBnUNJ+/dfp0EAAAAAAAAAAAAAAAALAsQo24hFAjopiOmG4abS5aGSdkx6o3vmy2n//i9N96AQAAAAAAAAAAAAAAAMsg1IhLCDUiyBRmVLP9lCOmy0C4EQAAAAAAAAAAAAAAAAsi1IhLCDXiKIQZy0e4EQAAAAAAAAAAAAAAAAsg1IhLCDXi3v76P3/7mDBjPQg3AgAAAAAAAAAAAAAAYEaEGnEJoUYc7C8nvz15sNUXF62IE6ozhhv12S9OvzoVAAAAAAAAAAAAAAAAEAehRlxCqBF3+uvJx62228cX7UcrVK9R0/+saT7/+elTLwAAAAAAAAAAAAAAACAMoUZcQqgRN9qefPr2f2z//sVW242AK5qtnvzswYMvCTcCAAAAAAAAAAAAAAAgAKFGXPJAwDX+dvK/Pvnb9j/OCTTiJttGn/64fXn2l5N/2QgAAAAAAAAAAAAAAAAAIqBSIy6xo6ab7csvttIjAQdrTt9oms+o2ggAAAAAAAAAAAAAAIB7olIjAAAAAAAAAAAAAAAAAABIwjbgOheKw/HTAAAAAAAAAAAAAAAAAAAgCYQaAQAAAAAAAAAAAAAAAABAEgg1AgAAAAAAAAAAAAAAAACAJBBqBAAAAAAAAAAAAAAAAAAASSDUCAAAAAAAAAAAAAAAAAAAkkCoEQAAAAAAAAAAAAAAAAAAJIFQIwAAAAAAAAAAAAAAAAAASAKhRgAAAAAAAAAAAAAAAAAAkARCjQAAAAAAAAAAAAAAAAAAIAmEGgEAAAAAAAAAAAAAAAAAQBIINQIAAAAAAAAAAAAAAAAAgCS8IQAAkLK3X13u1aW9n+/s/XVOd3vx6jJ/2vtzfu9/83t/DYC83dR+/PLVn9//a27zYu/6sy63Ff7VBeBmb+vy87j/x/d9Hs11/bnX5b7cCwAAAAAAAAAAIFOEGgEAWJ+7uB69+vnO3h8fGm6Yw3ONwQj7aYEJ/+p6LgCpcRdXq10bMrUnS7YfXrs24k+vfk7tCFADp9f78ulaqy+f+nH7+Uddfk55NoHlvH3Ddd1fY/yV/20/yOyv/DGAMsRsJ/Y3OQBYxv7z6a783Df9uev6cn/l1/T3wLLcDT+v+2t4hgEgfe6Gn9f9NVfbbN6vgIQ0AgAAS5kqNLUX16+1TvAohucaB/C/F8ElYGlOuzakVfptyH57MYhgNPJnz9ujV1fOffkUeNzvy70AHMPpcqh5aiemsf9cvHbPLRuRgHRNbcHULvxal6s3O81nfwFu+jltQJrGAgBuNwUWb+rrrwshx+T1+jPsr/x5ALdzuvs5novXrj+2fvfPutwP8wwDwP05XZ532d9gPme7fte7FW16HFsdz19cvxKKQqgRAID52MC5vbh+o10AYs5JkjVNgYhvRTACiMnajBON7ciJ8m9D7MV+0NhW2E8vIG1Ou7681bzBg7V5Xe7LCToAlzmN4/nf6HKQMVXThPoUXvbiuQbmtL+J8R3lsQHJTO3DH0VbATjtNhHar1Pv6820iG7XH8U4HnXLcRPi/jM8hWMGAYjJ6fKmoun6pV5vHz4X89Upya1dp02Pg1AjLiHUCABAXO3F9b7GwXWreu2HHAexQwm4D3dxfaixDWlVtkFjO3EqJoyQhtKCxCG8xmd0qrTqBdRjf+J8CjWX0B5ME+zTcz0IwLGcyt34cLWtoOoISjSN+6cF8tI2Ig+6XJndCyjPNP++f5pJKaa59f3TFQDcbAotTuG36df3rdr3nnhPXlOp7foUbmRcdhhCjbiEUCMAAGEIPxxmuLi+FqEI4CbWdmw0hqJb1WnQ2E5YwJFFUyyp1a4ffyTcZBAhZJSt1S6cVFqw4TaDdhuRWCwFbua0C0DV+O4/6PLGRSA3+yFG++lUF6/LfT7v3MiR024evlVdfbEXmw6BfU67fr1VvH6dUOOynOps1ykKcztCjbiEUCMAAPc3TYRaJbWaFjxjGUTAMTd2j3+h8n2k5bUX12PVXdn1Ov3F9aUIV2A+rcYQ8Ub048ewZ9Oe0UH05cgXm5Ne53U5wFw7e9f7RMf7TCxO5K7VOF6oMQB1G7uvB+0W4byANFk7Pm0cbIV9g9jUgDy0oi++zlT1i80G5dpofFc9VmnHKE8FAebeYESocX6taNevGsSa6T5CjQAAAEdqL65nF9d3GgdVXOHXmQiU5MApr/vq2Gspdr9/enH9Qfl/ZnNfZxrbCCCGVmOImH6c5xT1mhZC7L6lLbj9ss/nmcaFhlq1CvsMnZAbayNa8d5/3+tMY9vqBKyv1TjmP1dez9Ga17nGjaxUrUcqWo33JH3x4c/wMxHeLk2vsPuiVf5ajX36mZZ7nlphDq1o1w+9bM3I1o6c6hXy+Z0LAACgMrao8VgMtue+pkVTJlDT5JTX/XTsNTfak+OvcxGawnGmEPGZ8rrnc31On4lAA9LUiiBjjOe7trF6q7DPzQm5aMUiW6zrTIzbsbzpXftceT0vKV7nIqSMdbSiL471DD8Tc+wl6BV2L7TK09p9eivE0op2PfQ6U51FYUI+s3MBAABUor24vlG6g9mSL9uJtBFS4pTXPXTsNafH4gU+xnUu2gccphUTZ2tez8REMNbHZoJ5rnPVE3ZoFfZZOSFlhKDmvdi4iLnZM2zVhM+U17OR03Um3r8xLzYhzntNc+xOyFGvsO+/VT72T1RY+7lphRC06/Ndz1TPu1XI53QuFKcRkve3k399tNVLjuUEXmn04MWbp//7uYB5tBoXNlphbf7i+vziGl79GutxquNlYI6xcSsql83BX1wfXFyMB3BVK/rxlHiNfXkvYDmtaAeW8OLiOtX4jHuVqdW4GHOsX4n3mBS1oo1Y2nBxfS3GA4jD1kk+0bhgzprJMvzF9eTi+lb0a4ijvbjeV53Vp9bSq+xxe4n6i+tDHe89jWOwlKXYp+fwuaXIwnZ2v25Euz43Ww/5UmW/W211PK9xLgYFIdSYuL+e/PbxxWPbCcAlzYOfbd785v98LSCejcYXKKoIpMdrfJFk4mU9ToQa78uJSmVL6EXbgFErAgop8yLciHlNlR1sEp3x/PIs3GiT6oPK0opQY0k2GtuIVliLFxsXcbxWjPdT0It3cByvFc/x2gax0SAXvcoNNbZKty0g1Hg/rWjX1+JV7rsVoUZc8kBI1t8++P8+JNAIXG/78scnVsVUQLiNxrBWTaW7c+N0+XtyAtJmAWk74qUV5rbRGDb4VKjVRmP/YPdBK6TKaezD7bvaCIhn//hYO3Ke8fw6puM/7doISMf+MfRsOFqf0/g9nIl3exyu1a6PaYW1bcT8HO5vI57jVLTi3RzraUVbUIpWfJdrc9q9Wz0W4zIUjFBjov528i+Pti9/eCIAN3lb2x+++fvJx07AcVoxkZ6jjcawGIN0pMhpbFdsDMcxC8txGoMsZ6JdqEmrsT+gH8+L0/id/UF8bwizH2bsRL+bilYskiINtBFpcyIYhbu1YrE8ZRvxDONuG+3uk1ZIiRPjdiynFX16KVrxXabGaXznJdyIYhFqTNBPIa1t843UMOEG3GJ70TH/uH159t3JpzwruI9WDLpzZ898JyrBIC0fiuqMa2s1fgdUbSxbq10/TkW2fNl3xyIojkVQKX1Ou4oBtNVYmlVNp43Ix0YswOGyVszb5WQjxvV4XSvui1w4EW7EfJzo00vRiu8ydU67dVPWR1AUQo2J2Z58+raFtLYM9IGD2LPyxvbv3wi4my1mTJW8WqEETrtJlxMB67G2pReLpimY2nomzsvjtAvItEIpNtpVYAbuYuM9gkp5aUVVXSyn1dhGUDU9P05sXMQYgmexPF8bEWLD5dCLE3LitJtnbwWE2a+a3go5a8X4LDdO4/oIYXUUoxGS8rf3/9ezrbYbAbifrbq3/v2rzwVczyo1dGJho3T9xWXtgBdicxpfgkp337Gxu7gsWE8FojT5i+s90SaUwCZCbYcp/XjZvHhmcb1WYzvQKm8vbrhu87Z2bd/0a6d8WdjsS+XznLcaF2+O9SvRpi2hVRlthPFXfh7STrgrv95vN3LlL64PLq7nQg2m4EMJFWX8q5819/XGa5yf64VaOI0Bitw3nV/37Po7/p4Sn2HTi3n2tfUaTwY6ls2vDFqetQMWkM11PLrW55aaEsdn08/a3rHsncrerbzysdXxvMa5GBSEUGNC/nry28cWzBKAozQPfrZ585v/87WAnUcaJ1RaoRb2MmITLk+EmJwINV7lxM7zXHwm2oRctRr7cYLDdbHn1fryuyYYUb7cJtG9xsniP7369XS9UPzJ42mxdPpp169f/fEjpT3Z7pVP0KEVocaU5dhGWHtwUzsRu99z2i2+Te3Cr7VrM3JYlOtFoKJ0OW1Cnvrz/Wd4eqYl+vrreLFpqQY5bUL0uv4Znnu8Pj2377z6c7k8w6bT2Bdjeb3yCjU6jWHGVnkj1FjO+GyJdyz74/22PWVPlM8mU0KNuIRQYyL+cvLbk2YrjtAFwrx4o3nw7s9Pn3oB44RKJ9TKi4nTmJwINe6bjsXKZQISTMLmpqTdwDiOF/147TYaQ80p9rXTBPkfX/0clN69Ok2w2/UbpTnBPlxcHynt57wVocZUnSjtCjDPtWsnBu0W11IyLcLZz1/v/XFqvKj4VqLUNyF7jc/uH7VbKPdKy35fv/8cp6gXAeUStUr3uPGr4/XpGU6pL87pGfaiL15Dr3xCjamPzY3XLgBnP/+s6yuyDqq3v2J8Fm7/Hes3Si/E7pVHe06oEZcQakzA308+dj9uX55tqfQDBLto1Pz3zT+/+/D0SWqTxViODRKfiapOGHUiyBSDE6HGiU0mPRGBxhydagxPMEZIW6t0F0awvE7047VxSq+6g9c4cf575b3AMS2c2oKTLZq2SkOndJ/zVoQaU+OUXhsxBSe+1S44ket402lsJ1rtFuFSYZ9rbsem4XXWF03Vf1IyaBdCtivXZ3jq61uNz3CrdHgRiipFipsQ7ZkdtBuvP1eeUn6GTS8CykvqlUeo0UJwqbQH+1X7Ug6+pSbV8Zl9d1O7blfu71jTXEwK71i90m7PCTXiEkKNCfjb+7/9w5bwDRBNo6Z/89vffSTUKKey6FiOF9WeQjkRajQ2kdQLOfOiPUhZShOhSIcXz20tUhnL7weULBDvVSb7nFuNE+tTNce1DEqzamMrQo0pSel9f9DlBbZSpdROTDqx4SFXtv5hJ1U5rc9r7OcH5b1IfohWu2eYRXSEapXOJsRBuw0Fg8rVKq1n2IuA8lJ6pR1qdBr79TXvS6/LGxNyDTSvKaXxmY3HbA7m969+ljo+cxrb9vdf/Vzr/dYr3facUCMuIdS4sr+e/PbxxWPZCUBULxt99svTr54ItXBKr1oD0mIvQJ+JCZdjORFqJNBYDi8CUqlxWn8iFGnzYuGkZDaBa21Aq/VMk+dfK+8qayGsDd5onFh3Wl6K4/VWhBpTkEIbYQaN4Yle9Vb+XrudmAxK//h6XGah5LXnaQfVEUa+jdPYln6oddtUr/EZHoRcpFCdcdp4ZOP1ksMut3FK4xk2vQgoz61XuqHGtYJwtWxAXIq1653WZd/p1K4PqlOr8R1rrU1kNka39jylfpVQIy4h1Liiv5583Gr7MmRyFMDNXrzRPHj356dPvVC6VhxTicOlOEDPgVPdoUabqLExWwpVYRCHF8HGVHCkO+6jE9WZStNqXAxZow2YFkQ+V71BxpusGVxKabzeilDj2lqt+74/iCDjTaydsHDLWotvXmx4yIHTupuQ7bn9UuNCOdWbLnNaPxzVibF9Dtau4jWIvvg6TuOz+1jrfTdeBJTn1CvNUOPSG/8JNMfntP74rPYg401a7eZilpwn80prrYRQIy4h1LiSv5987H7cvjzbEsIB5vT8rW+/elcoWQo7iZAfL8JM9+VUb6jRXVx/EIGrEnnRFqyN46ZxjEFUZyrFWm3AVNnBwnMsiNyt1TipHrKgdl9eafTRrQg1rmmtNoIQ1P21Wr6dmFhb/pmQohONC+ZLv0vbMzxofI4H4RBO4/zqGiFlL97LU2ZVVjut8xzbM8x4/TDTRoOlQzCTTgSU59ArvVDjkutxgwg0z2HNzeWDduMzvtO7bbT85hN7r1q7uroh1IhLHgir+HG7fUygEZjdo7/9z99+IZRoOn6qE3B/TmNIbSPgdk5UaCyZ0/j9OmFpTmM7TKARx2jFs5s7p3XagEHjwo5tfOvEJPqhBo3jZpsUtglur/k50U/UzGls59doIz7QeK93ItB4H4N27cTSGw/sPrENeE5Iic3HLl2J2fp1C9XYfWjP8iAcyuvyMzxoOU5jm38ipMSeXQslLx18GTSO1x+K8fp92Jhlo3X6YdNpHLs7oWRLBBqnQPN7ry6CzXHZ+KzXOuMzm4ex7/RUfKeH6jV+Zta2f61lfPHqApJCpcYV/OXkXzbNtnkmAMtoHrz31unTQSiFEwvZiKcTO0kP4VRnpUYWx+rgRWWIJa19dBXKYZOgLFjnp9WyIYdpUaQX7XxMGy131F2n9cbrrajUuLRWyx83PWi8xwYhpo2WrSziNQavT4U1OS1/nOGgcaG3F2JyGvvgJSuwdmKOLgVOy8+9W9tNddW4Wo3j9VbL8eI46ph6pVOpce5AI9VZ5+U0zsM80nL4TuNzGjeBfKL5+2ivdddLqNSIS6jUuDA7dvrBtnksAItpti+ffXfyKVW2ytCKXX+Iq9Py1QOQB9uR5oQaOFGRcyk2GcvGBMRiz+walbxwPJt4Xaq93a/W1IlgWWy9lqsE04m+oxZTG+G0jF7jfRxzwRc7vXZVdgbNz2l8t2fefT22UG7PcKtlDNrdY70Qm9eu8ttS1YE6MUe3NntnX3LuvRfVVecyaPkKX05jP0BfXJY5A41X39sJv8XXanwulwo0eo3zBHyn8XmNIdEl5mKclr1vgFsRalwYx04Dy7Nn7p9e/icvUvlbchEUdbHdTYRlsc/aG0IydXEKq8SEu9lYrBf9OOKzEDpj/bTtH183NxZFltVrmQn1VgQbS2dt+RJthOm13vGMNRq0bKiiE0emrWHJzUuDlg3M1s5r2XAjc3TrWfKdvRd98VK81gko0xeXYa5AI+/ty1hy05h9h1OYsRff6dx6zd+POo1jMtbJsDpCjQuyY6e32m4EYHHbRp/+9eTjVsiVvTwttcCBOjmxUIqRE+1NrWznIZOu85j7mBqgE8HGVDmNY6yN5teLRZG19FpmQp1KAeVZsuruoDEERYBiHV5jX2DfwXPNy+4nW4BjM80ylgpCeRFmXJPXcsEoJ+boljRtQOo0v+Hielf0xWvw2vXDXvOb+mIn5GquuTw7kpj39vkttaa6H1DthaX1Gj97+w685sFmcqyOUONCOHYaSMD2JUGFPC01qQI4jZMtLJTWjWp9dftU7D6MjX4cS+lEMDk1TsuE0Abtgkosiqyr1zih/pnmmVB3Gu+pD4USOI3vX63m5UUQKiWDlgm0WN9DmGJ+NvbqNK/9xfJBWJvXLtx4qvk4LdNH1G7aXLDRvLx2ffHcwXbcbtByVTKtLyagnCer8NcprkFjG2Dzrry3z2fJoDoB1XR0Gp+vuTaedGLOFSsi1LgQjp0GkvDor//zt52Qi2nwvRGwnGkyj4XSOtkGFCfUzl7QWyGUtaffiH4cy7LJ8WdCCpYIk/iL6wMRVEqRVYSYa0Ld+pdeVArIndP8i9wEodLWa1dVZC5OhCnmMo31594Q1mu3WI60eI3jsDmDUczRzctp/s3d1hfbZhf64vT0mr8fNk5UW8+NfVcxK/xN7QDv7fNzWiaoPoiAaoq8dhtPvOJjzhWrIdS4AKvSyLHTQCIafWLPpJC6pXaJAteZFkqZNK1PJ2BkL+hOONbUj58IWN5GTLKtzcZQ1gbMeQylVQSwSl9zVghCGK/xeXxX80yodyLYmKslQs+DxnuvE1LXab6FN+NEsDG2Jcb6XlRhzkWv8bv6UvPpxYkKsTnN3zYOGvvimOEoxNdp/tCpE3NEuXAaNy3EMoh2YClO8weICajmwWu+irwbje/yc873Aa8h1LiAH7cvOcYQSMfbP2y3lEhO2zQ5yu49rK0XC6VArZwIRR2Lfhwp2IhneC0WaOw13wSnFxUBcvNc81WB6cR4PTfTMYRztRHWLkwVXL2QC6/d0fVztO1OBBtjcZq/stu0cWEQcuE1js3mbHttPp8+P465NxfQF+fHaxck95rHUhV+ESbmeGkKv3lhbk4E1fG6XvOcoDH3Oz3wGkKNM/vLyb9sOHYaSM325K8nH7dCighCIDWdmDQFatWKydb7oh9HSjYaFz+xHBsz9ZoPIYe8dZqnGlsnxuu5sOo8cy5+WOXWX4kKrjmzRdK5qrs6EWwM5TTvZ+jFxoXcDRqf4bmqNnaizw81dxCBvjhvvcZ2eM7vj4Byuux7cQrnRfhtSVO77jSP/eqMXsiN1zg/Gju0TrARiyLUOCM74vbBtmFwBqRou+XZTA9BCKSqE5MtQK1iTejVwirj0Y8jJbYoTh++DPucO83Di5BDKbzmqdrYiWc9dVbF1arzzLHoMS20fSDaiBJ4zVfd1Ylg47Gc5v3s2LhQDmuHbcw2V8W3TmOfgvuzz22uIyPpi8vhNX6Pn2k+nRi7p2ajOO/zFoi1/vy5sIS5A42DCKiWolf80DrBRiyGUOOMftDLD6nSCKRq21olVSEVBBqRuk5UbANqZP0TR9gexj6nEwHp6cSCydzmDDQOIuRQok7xKz104llP1XQs/RxswZSFtjJ1mqcijBPBxvtymu8zm46pZeNCeXqNz/AcwZZeBBvvi74Y92Xf5xxV1iedGLunwinOd2EbUgg2L2fuQJltOKE6Y1m8xmc05uax6T4EZkWocSZWpVHb2Sb1AUTQbJsvvjv5lB0E6yPQiFzY8RhMmgL1aUWo+S42+bkRkK5O9OFzmTPQOB1xxKJImQaN3++geDrRZ6fG3vPnCjmw0Fa+QfOEopyoKnIop/kCjVMQKmbFGKTFa/yO56i8an0Lc8mHsc+p1zzoi8vmxZHyNQg9pcbe1606bycsxWm+sax9n9NJGShTp7ihdRtnUBQCsyLUOJMfOdoWyMHbb7z8TwZm6yLQiNz0ohIZUCOOob7ZnIEmICYWPuP7RPM8/9MkOtVeyuc1ftcxww5sRErHnNVDLPRMZbc6eM0TinIi2HgXp/kCjQSh6tJpDLzEbLOnOWUn3GbOCkr0xXWYjpSf6zjqToSn1rRR2CZlr7E/74WlOM03hp02nAxC6bziHke90TgXA8yCUOMMrErjVtuNAKSv0SdUa1yV7d5gcRm54b4F6sMx1NebK9AEzMGe42/EwmcsFhqbI3ToxSR6jTrFDTsQYl6f0zyLbV4ccVmrTvEDFdZOsPh2vTnHTQSh6tRrbL+94iHYeDsn+mLEM+dx1GxKWodTWKVMr3kqauNmTvP1e1+LDSe18Yp7HLWN7yn6hlkQapwBVRqBrFCtcT32skrFO+SIUARQp/bVhdGcx0kCc3Ea+3A2NYWZ6wg72yEee7Eb+egV7/tnvL4up3lCFLZgysJp3WzsafdAzDDcRiy+XWeOzZxUYoZX/MCE09jn4zIn+mLE5zXf99+LTUlLCzmVxosA3NKc5gs0WqhtIzac1KrTuOkoxvffiZA6ZkCoMTKqNAIZolrjGuyFiTApcubEbnCgRix4jpxYOEK+qMoUxmmeI+zsGErbIc4ket284i2UOnG07Bqc5nlPGsTCKUaD4gfgO7Hpdt8cm5C9qMSMkVf8e4Hx/WVO9MWYj9f4DH+p+JhrX461mxsdZzqi2AtLmXPTnp2Y0Am1mzaPeYXrRUgdkRFqjIwqjUCWqNa4LI6qRCmcqPYE1KbV8ZN+peCIL5RgIzbYHMNpnpCYVQXg+8DEa5xM/1rhnAjhL22OxTZbNI9dnQ9584ofrLHKhE6YYxOyF0EoXDZV7YzR108+FePJCX0xlmDPW6wjSyfMNy3n2HckLzYjrsGC+3NU0LZwai9gNFVj9grHyRmIilBjRFRpBDJGtcaltOKYGZSF3eBAfWyhr+YxQ8jxNEBKrP9uhUPNtcBEVQBcxxZYNoqzUNqK8fpS5lhsI/SMm3jFDcpNFXBqHufPsQmZak64zUZxg432rlp7ZSD6YiypU/xgoxP98dw2Ou693otNCmuwvm2juLzG8dkcR8kjb15xTs9woi1HRIQaI6JKI5A1qjXOz2nc+Q6UZiOOpAVq4lTvhP4clVuANVGV6XC2QOoUlwUaewE36xRnodT6rg+FOc0xRrDvvhNwM6+4i+sWBKr13d4p/vM2iMpuuNtG8YKNtYeT6Yuxhk7je11MFBGY1zFjHS8CjWuYY8OJF98lbuc13iODwtT8boXI3hCi2TbbPyn+rhQAC2kaJtlmxNEBKF13cf1e4QN9AHmwSSWrPFzT2MGJxQyUx2kMNr4n3CZ2ZQBrOz8TgUYcpnv1M3Qy3PptG697IbYTxR8jEKLAobzGfjzWnJMFgqytOFU9nMbPL2YQzEJqGwGH2bz6GWMDgtM4ZvhMdaEvxpr6Vz9jFrTYaOzjWXOP65jTV+z9nRDc8iwQFvvUOy++Sxxmeu57hY3P7N3K8lOc4AgAAJJnL7RbLq7Cr+9UbnDXKa/vgotriWujeriL61x5fT9cXPe5qEB6M1sgjflZ23ip9mMBcZxO4fffH3S/0E4b+M9zKp9T/DFCJ+D+nOLdiyW/21/H2saYz3Av4Di94t2HJ6qH09hu0RdjbfaeF/terOlZvqpX2GfXXvn9nI4bKzFfsjyn+O9Y56LoDI7TK+zeO+bdKvReBwAAuJfHijv45uJK+brvQmkunPL6Hri4lrjOVY9nyuu74eI65iJo9zqn+BPpNS9KIVyn8HvwPkfZtYH/LKfynStuG9EJOJ5TvDDFmerwWHGf4T8IOJ7Np8UK2X6nOo6htv/Gc9EXIx2t4t6PtW002Ncr7LNrr/x+j4/4PTphDbE3nJyLQCPC9Aq/B+8zLgv9ZwEAABzMFoe3XFyVXfdZKM2FU17fARfXUler8tlR2zl9J1xcx17nqmPh8z7sM4n5GW8EhOsUfi+2h/2jCDXewd57YrYRnYBwMatElV6ZaKO4z3CpmzyxrJghvW9UvmeiL0Z6NqJ/iaFX2OfW7v1ex7StZ8IaHivu82PfuxMQJsbGk/usm4be8wAAAAdx4qhKrnqv0qoQOeX1+XNxLXWdqWxO8Y8O4uJK+SpxY8KxYoeVOLIKMXUKux/PddjCaBv4z3Eq10Zx24gnAuLZKM59WXKlN6e4c3bnYsEc8TjFew9tVa7YGxA7AfFsFPf+rPFdvVe89m9zz7/3XPTra9go7nPD94iYYgQb2wP/WaH3PQAAwEGeKe4AnIsrp6u0ozGc8vr8ubiWvEo+sjb2cSdcXDlcrbBR3M+0ExCfheBC7stDqje1gf8MpzI5xd30UEMlLSyvE/fnbWKO88/FgjniaxXv/iwxnOwU7xm2i80FmEOnuPdpaUUE7tIr7PNq936vc/FZp84p7oYTe18rec4a63AKu0/t7z1kXBZy758LAADgABxVycVVVgU3p7w+ey6uJa9OZXqsvL4HLq5Y17nqPjrRKe5EeidgPmcKuz/vWqxrA39/pzKdizYXeThTnPu0VVkeK94zXNqGTqTFKn3HuE9LrPB2rnjPMZsLMKdO9DnH6hVn/LK5599HyHkdZ4r3rNhFMBVzcQrb5HjIuCzk3j8XAADAHZw4qpKLa7pKOWrRKa/PnYtrycv6vNI40Zcvef+ca6yWc6ax0vVt19mr6zzh/6YSrk71OlO8z5EFUswt9Piju46WbQN+b7ucyvNY8dqIcxGGwrzs+bb7LMa9WgqneM+wXSyYY2694tyrTuV4rLh9MZsLMLde8e7ZM9WjV9hn1b76fc7u8fecizZhDY8V7xmxqxMwr1Zx2qebhI5tAAAAbnWmuANwLq6cr1J2kDrl9blzcS19tSqLBZFy+vxTv6wvsNDNM41hd1v8dYozUWy/R/vq9+w0jsMIpMb5zpzqE7Pa+rlYDMEynMJCS7dVCWgDft+tymtHnGhnkZ9Wce7ZUjYsnivec9wJmF+scPKZytAq3jNsn6sTML/QjUil9sl36RX2ObUajx++z9/DZoXlOcV7NhifYUkhFbXPdfucYej4BgAA4EYcO83F9fp1pvw55fWZl3qda1fJzY4C6TQeIXLT1b26nr36+2JOIHJdvkqqRrZRXp99ipcFNZ5pnNx5pHU4jZPRvajqeOx1pro4xQvEnosFUizL2tqQ+7e94fdtA35Pu5zKYs92rDa2lsVopMHenULv2bsqu+bgseI9wxxNiSW1inPftspbrIDndK31roo6OcW7f2vZHNMrvM17do+//pmwhnPFa9fPBCwr5D3rtg2mIc/BuQAAAG7gxKI5F9dNV+6Ldk55fd65XzY5d6bxpXCq6BbTo1e/7xNRXTfmd1ZCRTIn+vJjrzONIeJUF4bs38v6onPl9bmufbWqx5nifW4skGINIVUCzm74PduA39Mup3I8Vrw2ohOwPHvOa753neI9w+eiGjOWFyOcfKa82eJ/rOeYzQVYw32rBpb8PB+iV9hnZHO/h278oor6Oh4r3jNxLr5DrCOkkMZN84ehzwIAAMC1nineAJyLq7Qr98CTU16fd47XmcZFslbLm46v7UXgKeTaKH8xF0lquGzSplN+7btNGPXieT/kOlcdi/YbxfvMOgHrCQk8XLe43wb8fnY5lcEpXiXXPwhYh1P4fZzze/254jzDhB6wphinT2yUJ6c4z7BdVFrFmkI2Ih0yfi9Jr7DP59k9/tpOWJpTvHesrRifYT1Ox9/LZzf8niHPwrkAAACusVG8wTfX7YMxO+LUJp8+ffW5txoHjdN1nbdf/W+PXv31J6/+/qlKG0fSLnM9U76c8vqsc7lSDUS1Cp84q/F6prw55fV5r3mdqZwKfhsRbrzr6lQ2p3j3wDMB67Ix1bHvNteFldojf6/pciqDPdsx2ohzsdiGdXWqc1ywUZxneKsyNnIhX63i9EU5sn/vWH0xsLZece7nUk5NuUmveP037UJ6nined0j1Xayt1fH37+aa3482DQAARGeDhC1X1Otc44uNvZBYGHGJF3T752w0vjATdJznapUnp7w+59SvM+VxLzgReLrPlftk6jPl9Xmv8f12KnfCfCOe9du++5IXSp4pzud0Lo6iRBqcjq8S0F35vdojf5/pcsrfRvHa042A9Z3r+Hv4mfJ7rp3ijfGo7oYUhFRlPlOe83KfKM4zTKVVpMLeG88V577+QuXqFeczYoyeno3ifX/PBKTh2DHauV6fTwx5Js4FAABwxWPN8zJV22UTS2caQ4xOaXAaX7CsOmTMUvg1X2fKk1Nen3Oq1zON4eEcbUTg6ZCrVZ5a5fU5L311qiestRHP+k33QImc4n1GTkA6jj3W7upif3vk71PSc3GuOG0EYSikotX9798z5TvOf6Y4z/C52LyANNh9eN85Svvrc61i5RSvL94ISIfNj24jXa3K1CveZ3Rb/47l2efO+AylCTk5o7vye9GuAQCAaJwIu4VeZxon1nJ4+bBjqy3gmMtnm+qV40SqU16fcWrXmcqYYHOKc2RbyVeuC/bnyutzXuo6U71BLeurzpXX9zXnVWpVk3PF+Xw6Aenpddz9/Gzv92iP/D2myylvsSpDnYvFNqTl0HmNnINQxinOM1xCe4aybHT4vWvv6JyoQCUvpOnYjUhXrzOVqVe8fvymayMs7bHifX+5Fk9AuY4NrF89JSfkuTgXAADAnmea52Wq9Cv3IyydqOQU+v3n9t075fUZp/Rdn6g8Tjz/N11/UH42yuszXuI6V7k7/e/DabnjjnK4nqksG8V7XoAUhRxr5179Hu2Rf//V3ydHTvHGe05AWpzu3qCbexDKPFOcZ7gTkJ4z3X7f2v+ee9jDKd543QlI013P8qHXRuXpFeezua1twLKc4hWJ6QSkqVP4PU3bBgAAonCa52Wq5Otc+VRlPNRGhJuOuTrlxSmvzzeFy6p/lF6RplNe38lSV27f+7ny+nznvkpYwI5tI+6T6XIqx7n4TFC+Vsfd198E/v0lPB9fKE4bwbHTSFWn6+/ZM5VR9aZVnGf4XECaWt18z5ayufSZ4jzHGwHpcooT8rJnv7S5nF5x2gDahnQ8E+Mz1OGYY6j3T8nh+QAAAFE80zwvUyVe5yr/JXEjAg/3uXKr1uiU1+e79neb8xFl99WKZ//qtVE+Nsrrs53zOhfVGW/jFK+CQs5XKeGcjeJ8Hp2A9PU67v52qjfU6BSnjTgXGwWQLrs390MUdr+WVGXf/ntiPMdOQLrOtLtXcz8V56pWcZ7hZwLSF+sY6k5l6RXnc2GcnganeN+fE5C2Y4+hfvbq7w9t3wAAAAg4HXiVNqF2iI0IOB16dcqHU16f7VrXucqo6nFfTjz3+1dOoadz5fXZznWdicncQ3XK67uNfeW2KeEm5wr/LM4F5OFqcOnQ65nqDTXaf3uMNnMjIG2dypy32SjOM/xMQNpajffqmcoLeNh/U4zxuhOQh2OqepX6vj7pFac/p49Pg33mMb47KuEjF3avHnOPuyP/vv3xDwAAAFUaD7jOVO/EkU0edMrr+1rjymmixSmvz3aN61x1TxY7xZmALOH6g/KwUV6f61wXk4H3Z1UUYhwPlevVKW8bxfkcnIB8HFv9ZXPk35fzc+IUp414JiB99j7uVJ5zhT/D9ns4AekrcWPpRnH64o2AfLSKc993KkevOJ9JKe8pOXOK872di+8O+bB3Lbtn73ufHxuG3H9OAABA5ZzmeZEq5bJF/pKOLArhRBWwu65OeXDK63Nd+rIQG1Xexs+AYON45XA/8F1xdG4IWzw8V17fd6wr9+oP5wr/DJ4JyM+Z7n+vH/P37F9O+bHnO0Zb6QRgDTYfFeMZ3gjAWmK8q58LyE9okKWE9/V9veL06de942BZz8T4DHXa6Lh2nDEQAAAI8kzzvEyVcJ2JxZvrdMrre1zyymWixSmvz3XJi0DjZcfuwCvtapW2Vnl9nnNcnwqhnOp93jvlaaPw//ZzMd5Fnlot31Y45cWp7jYSKMGZ4vT1ANbRqs4xCGBsTjHGqRCdytBrnneUjbAkpzjfWy4nAwFXnWmetox3mYq8IRzk7ycfux+222cCULUfmp9/8PD0yQvVy4mXnpt8LhZvbtJdXIPGQKwT9tlkjQVrOiFH/uJ67+KquV+4yj4L+0xqD3taFbtB6fpQdftI4+QwwniNz7tNTjnVxZ6hTvl5rHBfa/zugdwMGu/f2vvA23yicF70scBaWsXZXPWRAKwlxni9F+N15MnmFL9U+HNgY9onYr72JqfCkmK06+YDAXmytfNWQABCjQf6US/t6IZWAKr2M/1nq7oH/bEG4CWxl2N7oRiE2wwagw/faAz7YIeJljx5EWi8idfYLp6pXr9Wupzq3qBgEym9EItXnUFmp3F+YFA+NgoPn3qxEQN56y6u90WV7ZucKJz1s14A1hAjtN2L+S1gLU5x1iA/F5AvmyO3/szpeDbWt3FtL1z1rZjLXpJTnDnYXrxjIV/Dq6sVcKQHwkG22+Y3AlC95mXTql5OcRY5SuIvrnfFhO+hvMbP60th3zTRgnxM1Qi9cJNBdT/rKYe3a96gQFXleXjVGfLO7VmK8e/LAily58W7yE02ihN87gVgDU5xFs3p64H1UKURGOcVYvRFVGe/HlUalxVr3ojxGXJHJXgEIdR4sG0rAGiSrrw0t1ZUtNj3XISajmXHLfMidhkTLXn5TDz7h+hU7+fklK5WdbIQSyfMxcZFn6kurfIZG7cirARMqJB+PYLPQN4IQwF5cwoPJnvRF6MMvcL7o1ZUBbsOocZltQrXi/EZ8ucvrq8FHIlQ4wH+dvKvVmmFIA+AC9tH3518Wmt7wNHTOwQaw3WqL/xwm1ZMtOTCglG9cAgLDNS6C8/GCk7p2SjtwOVcrN/+VJhbr/oqoOVyX8XYPMECKUph4xOqNV7WiuAzkLtW4ejrgfVsFM7CAl5AGWLMJ7KedRlHTy9rozhzsIzPUIpOwJEINR7gpX5wAoDR2/+kfzjVp1WdIYjrTIFGXgDDWZUUyo7vcAR1+rx4+bqvQfXuwkvxCOoaq8L6i+sDYSkW8htUjxyeKac4VV96AeWgWuNlBJ+BvG0UPmfXizAUsKYYfXEvoByD4lRrpGDRDlUal/WJwvVifIZyeFGtEUci1HiAhqOnAVzyQ6v6cDTuiEBjfL0INk7sOWOiJW22WMvzf38Wcqrxc3NKi1OdFWGtj/HCkuwzr+WZd0r/udooHGEllIZqjTtOBJ+B3MWYs2OBEViPbXJ2CtOL916UJ8Z7KKd27AzCUh4pzmZ35mJQml7AEQg1HuTBrwUA/62prU2wkNVGINA4n14EGw3PWtp68dJ1rFqDA05paVUfm/wbhKV51TXxmnql5dCggxf9H8r0RDCtwrHYBqzHKfw5HsSYGVgTFZOB6/UKX4uJUS2vBF4En5dElUbgeoN478ARCDUeYrtN8eg4AGupr03gSNzd0ZUEGufTX1yfCe8LqWKCOEyNxzz+UmmpbSLXi+Pi12TP/KA6pFzRvFV4wJr+D6WycQmVyeKMDwYBWMtG4WgLgfU4hc+99yL4gnKFbpK2IgKtMAhLahWOkwVQKuYZcW+EGu/wt5N/tfASR0EC+G/bOGXDc1L70dNeY4VGL8zNAhC1D2hbpVfdDUwQx2DBgW9Vl5TGC071jV8+ENZWyzHUKS+SxBjHDwLK1atuTuHjg16Mk4E1UZEZyFurcASTUbIYm6Qp2lHfnOya7H5zCjNoPDkOKNEg7m/cE6HGO7zUD04AcMXfTz52qoMTO9ksFOGFpXRiMm4jpIbdY3H0qktKG6Nqm8DtxeRICrzq2Vme6jNG1RfgdoPqDu7GaLsYJwPrsVCyUxieYWBdoRWT7b13EFCuGNXVbQNA7cWLmCNbTozNpYTVUTqC1rgXQo13aOqraALgAD/qx1Z1aFU3Ow6ZF77lfaq6P/ffCCnpRaAjlkF1TbY7paOmqsteLM6mpJaj599XeiysFLpww3FHqEHNk+mh44NBjJOBNW0UbhCAtTiFrz8yXkcNThXG3otrXuv3Ysy+FLvXQjeOeVFFG+WrZb4YkRBqvMu2+bUA4IqtGqc61Hz0tE0KPRHWYIPZD1TvoLYVR1CnhHBUXL9XXZzW51TX5K3tZvZCKqwvr2GhzSm95yw0aOnF5h7UoVed7x1O4e0WFUSAdYX29Rbq9gKwltDgi41fQsNeQA4GhYfwa17n+qOwlBiV8AcB5bMxzCDgQIQa79JUX5IawDUaNe+ofE71Vmr0Go9Bxnr8xfWR6rURUsAiT3y9sLRW9fCi/05RLbtvUzuCOvTfh1A/ahHjSLscxWizCFIA64lx9HQvAGsKDSZbP0yVI9QitLp6jJMMcjUIS4lxigdzMagF1aZxMEKNd9luOX4awOvqaBta1es9MSmUApucq3VgyxHUaeiF2Lw4gnppNe1GZ+IvTbVUa0yp724VvmAzCKhHjeG80AW3XrwzA2tqFcaLYDKwJqfw55iKyahJrzA1H0HNCQzLaRXGvisvoA6DmFPAgQg13uK7k40NcqjUCOA12zqOho2xqyhHFojwQio61fl9tGIMsjYvFnnmUtOxJ07rq2XS1osgcspqqNbYKp2+OzTMPIjxMOoyqK7JdKfwBbfQajkAwoTO2Q0CsKZWYbx4jlGXGEeVpna6wlIINS6jVficEJXrUBs2aOAghBpv8U/6ZycAuN7b3518WnrgqFV9vDi2MjU2YVHrMdS1TrSkYhDmQlh0Oa3qCUhzX6WtluNdUwkRtwrDpCJqVNN9H9pWWZtOvwusx8b3rcLQ1wPrIpgM3F9o31VjEY8XohLaUmKclDMIqAvzCjgIocZbbPWSCkkAbvQ/9F8ltxGt6qwS956QokF1vtBxBPW6WOSZT007hNfuS2sKR7ObOX01TFSl8Mw5hVeJZVIRNarpvg9d0KWNANbVKowXi+bA2lqFYc4KNbIxaEhAz6mOE9j2UaVxOaEbxwZxYgbqM4jgNQ5AqPEWjV7WclQbgCN8X3Yb0ao+vXhpSJlVa6xtcEulxvV4scgzJ3uWa5lUWzvU+GvVoRd9eA4Gld+2prAhoVWYQUwook6D6rn3W4Xh6GlgXVR4A/LWKmyuwIvnGHWKMZ9Y23y7F5bgFB5q5B0LtWKjBu5EqPEW2zqrlAE40ANtS24jaqwQ97mQMq/6qnBZG8MGi3UMwtz+KMwtxrF0uWDyIx+lT9Jav732O0Jo0IHnCTWr4f63dsopzCAAa2LRHMhbqzCDgHqF9mGt6vInYQmtwg0C6sRJELgTocZbbOsrQw3gHrZqnMrVqi692LWWgyeqr3JQK6zh98LcOP5kfrWEor2Y+MtJr/Kt/ey1CjMIqFcNk+kxjkWjmiuwHqc4zzGA9YQWEyCYjJqFjtdrK+bB/OsyQu8rL74r1MvufeYYcCtCjbdo1LwjALjJy6bUSo2t6kOVxjzYwLa2ao21HB2bmkGYm1cd1nyfaFWHQciJ9eWDyrZmqDG0UqRNJHoB9aphMp0gBZA3gslA/lqFGQTUyyvsndXel53qQZ+/jFZhBgH1snaKUC9uRagRAI7UPNAvVabajrvtxeJtTmqr1ngiLM2LNmEJvKjOr5bd5xyVm5/SAzGt1hM6jqdtRu1qmEynwhuQt1ZhCCYD62oVZhAhJSC0L6tpvp13/Pk5hQdlGZ+hdjwDuBWhxltst1snALjBdlvsjq7aSvBTpTEvtVVrrG33aAr+KCzBC3OrYZOCF+GKHJV+vOuaVZapwAaE+73KZe8WIeMDKigA6wsdZ/AMA+sKfU8veZwCHGpQmJpORiIEPb8Y86+DgLoNAm5BqPEWTdixTQAK12hbahvhVI9eBGty9ER1aYUlDcJSvDAXpzreZQYhR15lP/9O6z1/VGADwg0qF0EKIH+hweRBANYUuglpEIBBYVrVwQtLaBXGNpwQPkXteA5wK0KNtyPUCOA2JbYRoZUbcsORlXmqbSK+tiPh10bliuXwojofpzpQVS5fHEEdX+g4nglEYFTys0DwGcibPcMhc5G86wLrcwozCICN1b2O51TH+r8XlhBa+ZONY8BoEHADQo0AgH01hae8GCTlrKZjw52wJBZ6luOFubSqwyDkqvS21ml5oeN4+j9gVPIRy6HVoWgngHU5hWHRHFhX6CakQQAmoRslnYA42DgGxMG7Cm5EqPEG35187AQAt9iWuZurplDjl0LOBtVTTSh08RGHeyGqVC3pz8JcQncJ54CqcnkbVLY1nkGOlQXi+aPK5BRmEIA1tQpDMBlYV+h4vdTxCXCM0D6tVfm8MDen8HViLwCGdxXciFAjAByvxFCjUz1OhdzVcnz426rjSIwU8OKEUjiVjwBW3rzKnrhdY6OQUxj6QGCn1Och9Ih6AOsK3TTBcwysi2peQDyhfZpT+dhMPj+nMCWfEgDcF88CbvSGAADYqaGyk7HBkRdyZ8HUT1QHm/gchLkx2bMsL8ylhsrLVsX2TMhZyYF9p+URdADiGVQejqgH8hcydrJFcy8Aa3IK4wVgMp3ecWzfWMM6GKebzI93LCCe6X3FCbiCUCMAYF8tx09/K5RgUNjkRU4INS7DC8hfLX15Lf+dyNNUZXnJRQQqsAHxeJX3nuEUhiMvgfXR1wN5YxMSEJfX8X1jDXNKhBrnF3of8Y4FXGZjHSfgCo6fBgBMpsXXGgxCKWoJqDphCV5A/mrpy4HUOS0ndBz/JwG4yqssVBEB8saiOZA/p+PRDwOvC+nbalgLI9Q4v3cUZhCAfcxP4lqEGgEAk1oqHtnL3CCUopZJPScswQvInxOAFCw5tiasBMRXWgAodNGWdgJYl1MYLwBrczoei/zA60LHp05AmNC5GIKnwGXMO+BahBoBAJNaKjv9XijJqeoQekQNDsNEAkrgBCAFS46tCSsB8XmVJeR94oUYJwNrcwpDXw+si01IQHxeYWop8oH5MBcDxMUzgWsRagQATJzqMAgl8apjgY3jZJfBYi1KQAgaSIPTcpzCeAG4yqssIe8TXgDW5hSGBUJgXQRfgPhCn4vS59u9MKcYVRpZiwAu8wKuQagRADBxqgOTQOUZVL63RbBxCUwkoAS0FUAafqnlOIXxAnBVae+NIYtuHHkJrO8dheFdF1iXUxieYeB1oc+FE3A8wupAfIR9cS1CjQCASegEaS54WShPLUeKE1QCcAjaCiANTssJGcczYQhcr6TnInRs4AVgbSHPMfNgwPqcwngBuCr0XXbJjYgoj1OYPwvAdbyAKwg1AgAmNYQgbCKXRdvyeNUh9EgD3M0LyJ8TgBQ4LYdjZYH4vMrhFMYLwNqcjsc8GLA+pzBeAK7jdTzm2hHCKYwXgOt4AVcQarzRf/GyD+AupbUTTuXj2Kwy1VJ1gOprAA5BWwGkYcln0el4zH0AN/MqA5Uagfw5Hc8LwNpCKsJ5AbiJ1/GYP0MI3rGAeVDFFK8h1HiDh6c9E/sAbtWUtwBYS6VGlMerDky0ALiLE4BULNlvU6kRmEcp7/yh7RFzpEDe2OALrI/xOjCPkPALc+0IQagRmIcXcAWhRgA4HqHG/HihVF7lY6IFwF1oJ4C0OC0j5Nkn6ADczKsMLLgBeXMKQzAZWB/v6sA8vI7Hc4kQ7ygM4zPgel7AFYQab7HloQFwi62akgadtbzAeaFUNVThDH1RBlA+JmSB+jiFYSIduFkpxx45haGdANZFtVUgf1RqBOYR2sc5AetgfAYAByLUCAAwtYQgeFEoVykLjgAAoBw5jLEZHwM34/kY8TkA66LaKpA/p+Mx5wncjHEq1uIUhnsXuJ4XcAWhxls0TeMFADdomqI6Vio1Inde5XMCgNs5AUjJEmNspzBMpAM3K+X5cDoebQQAAOuiLwZuFvp8cOIJ1kLbDlyPZwOvIdR4my0PDYCbbV8WtUuSSo3IHd8tAADA/TGGAm7G88FnAKSA46eBvBGaAuZDqBFrcQrD+Ay4Hs8GXkOo8RbbZktZdwA3e7ClY82LF0rG8wgAAFLjBCBnvGMASAGhRiBvHCEPAAAAHIlQ4y0aXhYA3KLR1gsAluMEALdzAlAbKksAuMs7Oh5hKAAAAKTKK4wTsDwvADdhDgKvIdR4iy2BJQC32OpnXsgJA6GyeQEAANSHyi8A5sR7NAAAAAAAWAJzEHgNocZbNTw0AG7U6EFJbYRT+WjTAQAAAAAAAAAAAAAAEkeo8RY/6sFzAcAN3jz937QRAAAAAACUiY1xAFJAVWYAAIB0hI7NAAD3QKjxVv/F5CWAazVMCAIAAAAAUDLmBQGkgIVzAACAdDA2A4AFEWq8xcPT/sWW4BKAa2zVeAEAAAAAgFKxWAUgBV5haMsAAADi8QIALIZQ4x0aNRwvC+B1W/1RZamhAoUTAAAAAACHIQgEoAS0ZQAAAACALBFqvEvz8k8CgCuaB1uvsnCsFnLHJD0AAAAAvC7kfZ/3LAAAAAAAsATmIPAaQo13aDhiFsA1tnpAFVcgLTUMdL0A4HZsUgDS4jU/rzBOAEr3Zx2PBQUAAACkyimMF7A8JwA3YQ4Cr3lDuNWbp189ufjxRACA3DmhZAx0AYBQIwAAMfGOAQAA1sZ4BAAAANWiUiMAwNQSgmASqFx8twAAIDU5jLEZQwE34/ngMwBSEDqecAKwJq8w9MUAkB4vAHNg3IPXEGoEABhCjcidU/lCjo0DUAcqNQJpWeKZ9ArD+Bgon9fxaCOA9THGBwDgek5h6GOxFicA12EOAq8h1AgAqAmDoXK9o/J9JwC4HZOxQFqo1AjkzQmGdgJYF5Uagfx5Ha+GOU/gWKHjVObRcCzuHQBYCKFGAIDxqsMjoVQ1LLT9SQBwOybUgLRQqRHI2y9VBq8wtBPAukLHEzzDQN4eCsBNQvs4L+A4bDoB5uEEXEGoEQAwqSEI4YRSOQEACDUC6fDKA5VfgJsRBBrxOQDrItQI5C/kOS5lkwUwB/o4rCW0AIUTgOs4AVcQagQATGoIQrBoW6a3VccEhhcA3I5QI5AOr+V4HY/KL8DNSnnH8ArjBGBNoWN85sKA9XkdzwnATUL6OC8AQGoIq+M1hBoBAJMaghAcP12mWr5XLwC4HaFGIB1/1nJCnn2CDsDNSnnPoMobkD+v47GBAVhfyLsB/TBws5Dnwws4nlcYJwDXYZ4SryHUCACYeJXPCSUirAoAO14AUuC1HK/jsUgKXK+kZyM01OgEYG1sYADy5nW8Wk6oAY7hBKzDKwzjM+B6jHnwGkKNAIDJn1Q+Gww5oTS/UR2eCwDuRrVGIA1eywmp/OIE4DpO5fAKw4IbsD6v4zkBWJtXGCcA13E63h8FHI+NY8A8nIArCDUCACZedWiF0jjVgaASgEN4AUiB13K8wjgBuMqpLF7HozI+sL6QjchUeQPWFzqnR18MvC70uWCuHSG8wjgBuI4TcAWhRgDApJaXuFqq+tXCqY6JPao0AjhUDZWXgRwsObb2CsMiKfC60p4Lr+M5AVibVxgnAGvyCkMwGXhd6HPBfDtCeIVxom0HrmJ+Etci1AgAmNTyEtcKJallkMvOUQCH8gKQgiXH1qH/LCcAVzmVhSpvQN6o8gbkzSvMrwXgKio1Yk0vFH4P8Y4FXOYEXINQIwBg4lUHJwZGJXlfdfijAOAwXgDWFmNy+z68wrBICrzuHZXFK4wTgDWFbmAg1Aisy94NvI7HMwy8zikMlRoRyitMKwD7GO/gWoQaAQCTpRdf13QilKJVHbwA4DBeANa29OJI6DieSUPgdRw/fRntBLAurzBOANYWMl53AnBVyOa8mtbCMJ/QQhROAPax6RrXItQIANjnVYdaqvuVzhbWnOrAzlEAh/ICsLY/a3lex3MCsM/eM0o7Cowqb0DeQqu8sUAIrC8k/GLjEicA+0LGp8y1IwavMLxjAZc5Adcg1AgA2FfLEbclLlLVaKN6sHMUwKFCFzwBhFtjgYRFUiAep/J4hSEQBazP63hOzIMBa2ODARBP6PrOGhsRUR6vMLxjAZcx1sG1CDUCAPbVskPNXng5gjp/tVTctIASu0cB3AdtBrCuNZ7B0H9mKwCTEifSQzc9sLgArC90IzLPMbAurzCtAEycwgwCwg0K48SmE2DSCrjBGwIAYMerHh9eXL2Qq1YcPQ0AN/mT6jCIqpRIU46hRoIOwM5vVCZrJ5yOY4ttj8S7CbCmGH39IABrGRSGil7ATqswjGkRg9e4eSwkmNheXKcCwLwkbkSoEQCwb1A9Wo0vGxzrm6cPVY9ajoUHEE8tk7PWPn4qACb0uWeRFNgpdTI9dNNDKxaAgTXFqMr8RADWMlVNdjoOi/3ATuj7K2NaxOIV1j7b30uoESh3cyki4PhpAMC+0COpckMQIk/u4tqoHkyyALivWtoNJjuAnRhHy3LsEVD2szAoDOFnYF1eYRg7A+sLeVefqiYDCKvU6EWhC8Tze4VhfAaMGOPgRoQaAQBX1RSg+kTIUau6EGoEcF/WbtQwQWuTHU4AJiGT6SySAqNW5YpR5Q3Aemx8TyAKyFto+KUVgFZhOBUJMYW+YzE2A8b5fSfgBhw/faTtyadv/03/RUcDFO4H/Y/nD0+f1LZryyZXTlQHm9DdXFy9kJPHqkfoogWAennVMTFmYxaO0QNGNmb4UMdrFV7JDchdyZUyvMKOvXSvLi8Aa7E5u5AxfivmGIA1cYw8EC507WoQEM+gMLZG2Yr7EnVrBdyCUOORXlz83xvb/+dMAEr24uG3Tx6qPrVNbtrCby/kYqO6duyw2ADgWKELnrl4XyzqAJPQcQPHHgHlT6bb+MDpeGwmANYV2tczdgbWxXgdCPdrhWG+HTF5jYUp3tbxWhFqRN3eF3ALjp8+0sPT/oWaZhCAYjVNc6o61XJk5aQVu0ByUlOVRvOtAOA4tUzStuJ4CmAyKGwc3ypsIh7IXavynwGOoAbyNigMJ08B64pxjHwroF5OYc+APYODgLgGhSGwjtrxjoJbEWoM0GhL0AAo2FbN16pTjcfdfiHkYKP6givsHAVwrEH12AjAJHTsEHqUF5CzGu7/QWFYcAPW5RW2gYFAFLC+3ytMK6BercIw1445xGjX2WCKWrWiYAHuQKgxwPf6R6+6qpkB1WjU+LdOnw6qV+ggPDe2C2QjpMypviqN7BwFEMK/umrwiQBMQjdfElhCzWo48ij0ZAYCUcD6Qvt6NjAA6wo9HYrxOmoWOl6nWBHmMCgc4zPUinsfdyLUGGA8gnpbayU3oGgvm+Zz1W1QfaxaI7uh0mWBFae6sHMUQKhaNikQsAB2qNQIHMepnveNQWFoJ4B1DQpTQ4AbSFnoeL0VFY1Qr1ZhmG/HHEI3jpkPBdSJzRq4E6HGYD8L3VUFIEE/Uh1tUH2VaC0Q8amQIqc6vxt2jgIINagetVXzBW4yiCpswDFqCuqFbnpgwQ1Y16AwTvT1wJpinMzCBgPUyO77kKIUXqz7YT6hazl2mhxFV1Abp/HeB25FqDHQT8fTNs0gAMVomqZ/ePrUCzXuWrNABAOo9JypTmycABCqpnakFYuzwIRjKYH7qymoFzo+IPwMrMu/ukK0ArCm0PE6FVdRo9D7fhAwn0Fh7B2LuRjUhnseByHUGEX1x9QCRfmeZ3pSa5W4L4SUWNDUqT5e4YsUAGAVILzqQbVGYDQojIW7qBCAmjxSXZvbvMLHByw+AOsKnbP7RADWFLrBoBVHUKMuMQJfnIqEOcXYWE5FfNSGdxIchFBjBD9Va6zvmFagUM1Alcb/1qtO7cXVCSlwqve7YJIFQCw1tSetqDoDGJtMDz2CmurlqEmNE+mh4wPCz8C6qLgK5M0rfIPBRkA9Qo+eNoOA+dgczKAwHEGNmtj97gQcgFBjLI2+FIDsbZvma2ESYxCeK6v01Aprspe3Wo+dNhw9DSCW2toTqjUC4zj+ucLwLKEmreoTIxBF+BlYz6DwIgtUXAXWFboOQUUv1CT0fh9EcSLML3TjmL1jbQTUgSqNOBihxkh+0D+eiAERkLVGjf/F6dNe2FdztbhnYlfUmuwYcKc6ebFzFEA8g+p6T2nFxgTAhI7jWzEWRh02qvO9Y1D4+IDwM7CuGIEo+npgPYPCOPHuizo4hd/rFDPBEnqFe19AHVoBByLUGMnD0/4F1RqBvL1sms+Fq3rVy11c3whrsMWxjeo1CADiqm2TAhsTgHEcHxpY+lRA+WquchS6uNuK/hZYU4yKqxsBWMug8COoqXKEGsTYSDMImF+M0+9aEfZC+Tbi6GncA6HGiKjWCOSLKo03qvkIatOK6hNLs0XFTnVj5yiA2HrVxYkwFhDjCGoWSVE6p7oXjEIDUYb+FljPoPC1CKoBAetigwFwt1ZhbKOvF7CMGBvLTwSUjflG3Auhxoio1gjkiyqNt6r5CGrTiYWapTy6uJ6obl7sHAUQnwWbatt8ZZsSHgmoW+g7ji2QtgLKVfsGtkFxKkQRpgDWEyMQ1QrAWnqFsT6YeWuUbKPwal4xNvIAh+oVPgdrhT94x0KpWjFnj3si1BgZ1RqB/FCl8U69aNe+EJO8c3MX15l4WWNzBIA5WD9eYxXYb0S/grrFCDRTtRylcuLYVRM6PuD4WmBdMYIa9PXAerzCNzezwQAlC+2jvOo7vQTrinH6HYF1lOxDAfdEqDEyqjUC+aFK451qP4J6YsEIdo/Mw4lA44SdowDmUmP74sQibaqcxr6/FeYUI9Dciu8JZWIifRSjUj5HRwHrGRQ+Z9eK+S5gTaGnJNl8KkeVokQbhVdpHAQsL0ZOhMA6SuTEpkgcgVDjDKjWCOSDKo0HI6w9vkDY4jsTvXE5jZ+rEwaFH/8GADcZVOc7iu1sJriSlmlM1b76+UyMA+bUKxzhYJRoI5gYmxidCD8Da/q9whFOBtbTK/xdnWcYJYpxX1PQBGsYFN6uUxEfJWJ+EUch1DgDqjUC+aBK48EGjcfX1Y5gY1z2ORJo3GHsAGButbYztumMvjsdV0OMm4vrXOPEFrvQ47Mx/KAwrRivoSwbcU/vizE+YHECWE+MAgsb0S4Ca7HnN7QvtvfdVkA5WoXP4wyigADWE6taI1AKJ4K6OBKhxplQrRFIH1Ua7y306LpS2GL7HzRWfsLxWhFo3OfF0dMA5hfjiMkcWd/9jehzUmChl5uORus0jrE2QmwxNnIRWEIprE/gfr7M3kNC5zBbEaYA1mLPb+jxtYa2EVhPjDlBnmGU5JnCsZ6FNcXIiTgxR4ZyME7B0Qg1zsSqNW6bB58JQLK2TfORcB+9CGvv+0IMwo5lO8ws0Eg1ph2qxgJYQowjJnPlRN+zNhs3dXf8NU7j4sW5CKHGNChOBScqnqIE9i7ihKuo1gjkrVe4jWgfgbXEqq7eCsjfRuH9kVecvhE4ls3BxAjW8o6FEjgR0EUAQo0z+qkCXNMMApCcpmn6t06fDsJ9xDgKozSdxnAjAYnD2edVa6Ww2wwCgGXUHKJ2Iti4lkMCjfucxmDj1aOqcbwY4/gvBOTNiYn0m8SoJNKKzxdYy6A48wr09cB6Yryrx6huB6wtRoiLAgJIQa9wTvebTwNSRDgXQQg1zq5h4AQk6HuezWMRRnudHUNtRyU64TZOY5CEY7tf12vcPQoASxg0VoGolVWas/7ICUu5b6Bx30bjOIvJr3CxAkutgHxZW+KE68Q8vpbNA8A6Ysx1noi+HljLoPBwshNzr8jbRnGqNA4C1hejCq+x0wZ4x0KunNj8iECEGmf2UyW4RqcCkI6tPn94+tQLx4hVMr00TuOC+0a4Tqvx82mF6xCyBrC02vtygo3LCQk0Tt5+9XtY5caNcKxYVdcJmCJXTrQhd4mxidGJMAWwlkFxNkzS1wPriTFHyAYD5MopTh9kc15eQBpitOvWpvOOhVxRCR7BCDUu4Ac9+EzhFREARNCo8W/9+1edEIJqjdezF4tn4pjEffaZ2ICVoz5v1otJFgDL68X7iRPBxrnZmKhTPO7V78n3djyOl0XNjplI96pLzEoiTgDWEGPhvBV9PbCWQeEnKxB+Qa5ijSF7AekYxDsW6rXRWAkeCEKocQFWEe5lQxUmIAUvG46djiDWQkepNhoX2zeqW6uxOiOTaLejTQKwhlgV23LnRKXlOTjN+7m2Gqs2spHk/mJWa2TDCnKy0XET6TX2lbEqiVCNAbkorT/rFWfzEn09clHifRpj/EH4BblxirOO0IsCAkhPrHesZwLyckz13V7AFYQaF/LL06+eqGkGAVhN0zT9L06f9kIMHwm3cRpfMP6g+iaQnMZQJ1WU7taLSRYA64lRsa0E06Qg4Ys4Wo1jgEea3+bVP4sNFPcT49l34nNHPpyOm0gfFF4pKUeD4mxitBBpKyBtTuO8zRcqa/4ixsK5E8dQIw/2/Ja22alXeF9M+AW5OVM4LwoIIE2D4rxjteIdC/mwdwmn+6Mdx2sINS6o0cvPBGAVduz096JKY0T+4vpauIst6NdSSWiqxmH/va1wCNokAGuiWuNlFtCyPswJx7JxwNKbGpx244+NcIiY1RqXCK8CoZhIv79Y/+32HkylN6TsQ+2C+iWduGEbGLzC2efSCkhXq/G5tcveB0oKKMc6Sp4jH5GDWJVFbb3KC0gT71ioibu4Ot1fL9pxXINQ44LePP2352q2LBwCK7Bjp+0oeCGmTlR4OtRG5YYbncaFQvvvo2LP4XoxOAewPqo1XuY09mdUpbkfC7ZZlaM1xwFO4ziLI6kPE+vZp8IpUtfquJDSoDiVNHI1KM5/vxN9KtLldHmhzf7YxhGlbHKJtXBOX4+UXa1EWFJAeVCcvpjwC1LndFzw5Sqv8T0XSNVwcZ0qnBPvWEjfsdV3KQSDaxFqXNgP+r6zinECsBiOnZ6NFxWe7mujcYL8G+W/273VODC1/55OTJDdF4NzACmgWuP1OlH57xBTlWYLNKZSsW+j8qq0zMGe/VjVX9jUglQ5HX/sImN16SPFQaU3pOqmxWCnMjal9ooTiLIxXicgPRtd/4zanysloBxjPGLvbIRfkLJYwVt7Xti0i9TZiZ4x7lPesZCyY0/L6EUhGNyAUOPCHp72L9S8/EAAFsGx07OjwtNx7OiPKRB47ABvDdNEmIUX7N+/FY5hbZIXAKSBvvx6TuPkuvV5rXCVHQ+VcpXm0o6RnIM9+88VLqexLOpy7L05qO4qjROveBsfqBKF1LS6e4xg/7uNJXIO78eaD7X2NJUNLIBxujuoZ39N7gHlQXGqelk7xjHUSJE9x63CeY1hGCB1XrxjoWxOx2+IIsuBGxFqXMF4DDUPJrAEjp2eXawqL7Vy2lWDmo5tTG2i2P59pmDAdxr/fZnMPp4XR2EASAt9+e2szzsTYf7JRuO4xfqy1CdPnXZVWlrhOp8pnN0H3whIy0bHh5pjVSgsQac4Gx+cqBKFtBxaxdVprP5sY4kcA0GD4oW0ra9n4RypsA1W7sC/dqO8A8qxqnoRfkFqnOJVAo7xXgssJdbmcifesZCeY4+d7kUhGNyCUONK3jr9qlPTDAIwG46dXkysKi+1s9DEdITjdET1GmXkpxCjTXZ99+rf5wsRBoiFozAApMj6ci/cptWuyvJGdbHFLxsb5FrpxGn87nI/RnIOg+JUf5nGsUAKnI6/H3vRH+6LufGBKlFIxTFVXJ3GOZocxxKxgtpOLJwjDU73Dyg65RtQ9opT1YuNSEiJ3Y/HBl+u6hXnnRZYir1jxQrirrF+CNzk2NMyKLiAOzXCar47+di9sX1pYRF2SAGR/XTsdNO8R5XGxbSK9yKK61lw1F9cf3z16xev/tjr/pzGvufRq1+/s/dr+qT52Pf2rvLkNE7+lo6x8bI6lb8w1iufik+t6Mvvw/phmzj/WuUeU9peXO9rDHGWND7oNC4MsslgZN/tueJ8x++JY3uxLruPbZ7N6Ti/0uX3q1ZhfePV3y9X9pnGqNZv7a69D3kB63CK8177UHmNIzrFe+/aaBz/AmuxZ9gpTI5j1hj/3caCNE8ErMs2CWwUzmt8nr3K1F9cH+p4vJ+nLdaJMF5lPwfIw0aHV8O/ygKN3ZU/t9XxvMa5GBSEhduV/eXk402zfXnsQw7gBj8023cf2lHvWJJNiHwirMXf8Gt3w6+xvJwXNZ0INSK+ToQaU8MRy8fxGieKSwg4WiBoozHM2KpcXuOkWS8Y290fo9KiF5PpWJfdx8ceL3ndRHorQo2mVbyNDzZPY+0EwXKsIUYoqFd+x9SHBr73EU7Gmmz+oFMYrzwXulvF64vtGWbdBGuJ8RxPSg/p9iLUWDKncXwWY4PpoPH7BtbgNI5RnO7P6/pxGaFGXMLx0yv76WjcZhujfDyAyVafE2hcRScmNdfk9q5279r/81iPLZJ6AUDabIGWkMH9OY1BQJvA+U7jsV4WqolR1WoJrcaFhenf/wuVH251GncQx6p4kjtbCBoUzun4ndlAKGvHjg00elG16DaD4hx9aaxv5AhbrOHY49CuyvFoNBvfxwpiTkeGcsoHlmb9R6dwOT7DZlC8vtjeV3mGsYZW8QKNXozfkTeveO16K96xsI7p3cDpOLmOy7AwQo0J+EHfd2JnFBBHo9O3/v2rTliDTZJ+JgBXecWbsAGAOXnFm1CrlU3mnGgMBtqOawsJnr36Y/vzawcdnXb/flOI0X52qrNKp9MYbHwmwo2xQs2t4lR9BO6jVdh42ybSCfXfrlO8TVqf6vgAKnCMVnHeyW2c7JWn4eI6VRxObGLAspzGIF4or7wrtXeK0wY5xfk8gftwitt3UJUOJegULx/SaZzvA5YUsnGsFyfo4ECEGhPw8LR/8UPz4AMxgQoEadT4H/SAUN26ThVvkhQoRW5HUwGoWycqy8ZkIcdWY3jDFo4s6Lh99XMKO3YaJx7tr7PQo9Px3tauavPm1e/9TLuA5bl2lSRbUaFjstH4GdW8s90r3g5pu79CjskC7sMpbIHU3l974S4xK72ZGqoCIw1OcUIUXvlXhIpZld3GrlQEwlJqrrS6L2Zf3IpnGMtxCqvkdRUnIqEkMcdnbNjFkkJOy7B7niqNOFgjJOOvJx+32r48E4BjWDj43YenT72wNlsct0VhJwBWyaGEKiROYxCmdIyNl9Wp/En0XnkGm1uNE85Ynz/wr3tbBBRj8Ron1nrVyZ79VuFsgtKqZ3AqBebkFL5A+ivd3Na2CusPb/u9c2Whrk8Uh7UT74pFacwrVr9mY/pe+bMwYswKbba5nOM/MSebM+gUrlc5m45j9sU8w1iCrRXFOrViUD1VGnuFbRa0z2kQcmDrR7FOvPAa37EopIU52TgkZPxw1/hjq+N5jXMxKAiVGhPy1unT4WIpnVQycITm4tkh0JiM2BUcgFx5cew0gDwN4hjqVLgDLwKN8TiNu9tjVpLISawqAXZPWmjCCZhHjHuMKi/31yneZ2bfYa1tLZZhYahW4bzK2exg1WljjvOpuoo5tYo3r1bSuluneH2xPcOxwmbAdezdOtY95sW6E8pk4a5BcTjF3cACXGVtekig0YsNFbgnQo2Jeev0q2671dcCcLitPn/z9Cs6wLQMIgwB2G5IdsQByFUngh6oW6uxSnFtx/d4xVsochoDS4RuMQdbqAlZIPViA9IxYm9idBq/S9oJxGbVQzrFUVpFqE5xx/mh7TFwHac4R8eb0jYxTBXRY805ssEAc7HNBRvFw4YklCzmMdSt4vWhwD6n8NBsLdV2ERGhxgT9+OAfVmaYI5qAA1gI+K1//6oTUkRbhpoxyQIgdzaR9oEAbDQu9H2qesSs4uREsBHx2QJNqzBMpB9vUNxNjBaGopoIYgqtHrLP7nWvssQOJ1OdGbE5xQvaeZW5icErXvVJKidjDrGOjp9Yf9wLKJdX3HnYjcbnEIjFKXy8wLopjkKoMUEPT/sXPzQPPmjUeAG40cUz8vzHB/9c0+JijmwQTqU61GYQVV8AlME2J3wmAE7j0Wznqmexr1O8DUoWLvlCQBwWaNwojPVtXghhczGD4mlFNRHE4RQvJOtV7rv9oLjjfCdCUYgjdki25E0MFt6OdeqbExuREE/sQKNXXZsMUa9BcTePdSLYiDicwsf6g1g3xZEINSbq4elT/33TcGwlcAML/V48Ix88PH3CM5I2r7i7v4HUeXHPAyiLLZQMAmAG1ROEmqq1xnrf2ojAEsLFCDRaJdJYFdxqZ+89XvFsRDuBME5xg3UxjwFMUexxvhPBRoSzjTCxjjOvoRqQBb284nAi2IhwcwQaqbCOmsTePNZdXB8KOJ5T+Bjfi3VTBCDUmDALNjbNlmAjcMWrQON79owIObBFo1jHYQCps8V/LwAoS+zQApAjr/rGtF4cf4R0xAg0elGBOCav+AsTGxFsxHGc4gbqrM8fVL7YcxhOBBtxvBh9/cSrjmpAtnYYcw3RAqUEG3Gs2IFGw3wUahR7fNaLYCOO4xRnbM+x0whCqDFxb57+2/Nt84AJV2DnBYHGLHWiyhPKZwPzWMc0AkBKvNhNCdR6XO2guCGwTgQbcX+xQg4sisY3KH5QdCOCjbgfp7hBOq96jkaLXZnZOBFsxP3Z+HCjeGqq7OYVdyMSwUYcY45AYy0bDICr5hif9SLYiPtxihdo7AUEINSYgV+cPu23zQMWEYGLAdwPzZZAY76oYIeSfal6Fj0A1GkQ1a1QL5uAO1W97HjKLxVPJwJLOFysQCOLovOxNuJrxbUR7QQO4xQ3QDdVPauJbc6MPc53ItiIw8UOQ9W4GWlQ3OeYYCPuY45AI3PtqN0c47NebDLFYZzijOXtPu4EBCLUmAkLNqrh+FbUrWkefPbw9N+ogpav2MdhAKnwYmAOoA5zhBaA1DEBN/pUcQNhm4vrG7FQipvZvWGT6BuF68VzPDdrI2LP12xEsBG3c4ofnKv1aLReir724ESwEXezdr5TPL3G99Yaxd6IZMHGP4hnGLebI9BoY8pPBaBX/PFZJ4KNuJ1TnDG8V9xK0qgYocaMvHX6VUewEbVqmgcfvWnhXuTOi2AjyuLFPQ2gLnOEFoBUeTEBty925fUTEXbA9ZzGe6NVOC8qDS9hOiLNK66NxkAFAWhc5RS/D7EwUK1hKNMp/gYmJ/p6XM/a9VjVmCde9Pn2vh7zOXbiGcbNYoeSjRfv4MC+TnED69PvSbAR15kqNTuF4/RGREOoMTMEG1EjAo3FmaNsOrCGqfqoFwDUY67QApAi7vXL5hj7xJwwRRmcxnvikcJ5sQFpSV7zfN5UisJVc/QdVIUazbGByWl8hmO06yhDzGrME04I2on9HDvFG5uhDPYMW7u+UVxezLUD14kdWDedOD0Dl8XceGwZAIoiIBpCjRki2IiaEGgsVi/RjiF7H4lJFgB18hrDXiwYoWRMwF3PK/5CkxMLpRjFDq8RTF6e19hGxOZEABqjDxX/XvCiKtRkrg1MUwCG4Cic5gm5Mke3M8dGJCeeYYyc5nmGvQg0AreZY+OJhdjYPAbzieKFXG3t/4mAiAg1ZopgI2pAoLF4nQg2Il8WdDgVANTLJtJY/EWpmIC7nVf8YLMTC6W1s0n0mMcMW7iBYPI67HP/SPE5jffIiVArOyavV9yKMpzA8Dqv+T6TL8RxhzVrNU94wsbuzNFdNlfbxjNct1bzPMOcCALcbWrX56iozSbTulnfHmsO1CqKdgIiI9SYMQs2vmw4whVFeqHmwXsEGqvQiWAj8kPQAQBGg8T7CIpjC6KdcBebSJ/jiL8vXl2oS8xJdGPj9V5YU695xggWZrMKEgQq6mLf+zPN0z9T3e16XvMFGztx3GGNbPOChSZif+/W53fCdbzmeY47UT25RnM9w3MFtYASzRUAdmKTaY2c4n7v1o5vBMyAUGPmfnn61ZNt82CO3c/AWl780Gzfe+v06SDUoru4vhSQByZLAeAyC6GwQQGl8Jqnulip5go22oTquVgorYHTuDgac/GE8Xo65hwjdBpDboSiyuc0LrZtFB8nMNzOK35l5gnHHdZjCiXPsTmYakB385on2NiKyl61mPMZJtAI3J/XvBW17eIdq3yt4vbjXpzohBkRaizAL06f9k2zfVfzTDAAi2nU+B+aB+8+PP03XmLqY4tYXwtIGwukAHC9TgQbkT+veQJ6pZsr2Og0TrBuhFK1GgMtreJhvJ6eTvONETYiFFW66Vh6p/g4geEwc/Xzxmm+wCrSYAvlc33Hp+LeOZTXPAEYp/H7pXpyueZ8hgk0Asfzmi/YaGu1vGOVbaq86xSH13z3I/ATQo2FePP0355bGMxCYQIyZPfu903z3sPTp16o1UYEG5EuFkgB4HadCDYiX15MwIWYM9hoVUGoFFAe+05jH1/HeD1dneYbIzhxVFqJ9qtCzdH+017cz5zBxum7pq8vT+wF8312T1Jd/X685j9S3gklmfMZJtAIhPOar113Gk/P4B2rLE5jux5zY5cX86lYAKHGglgYzEJhBBuRnaYZvm9+/i6BRohgI9LEggcAHKYTwUbkx4sJuBjmDDxQKaAcTvME0Bivp6/TfGMEC0JZIMqCUU7IXat5q/fRXhxnzn7e0NeXw2m3YD5HUHXue7FkXvO995yIKuulcJr3GSbQCMTjNe98Fu9Y5bB+OvZJGV7Mp2IhhBoLMwYb/++7an4qvw+kb9t8+dbp7957ePqEiQhMNiIQgXSw4AEA99OJfhz5sHeQD8QEXCxzH1FplQI43i5f0zGyjxQX4/V8dJp3jLARgYqcTeHUuapCGdqLMFM/7zUPJ/r63E19fat5EGgM5zVvZa9nIgCTszlCL/sINALxec37XG3EO1bOpqro3yhuUN2LQCMWRKixQA9P+xdvnX71gRoWE5G2ptFnb/377yhfjet0IhCB9bHgAQDH6UQ/jvSxoDIP+zzf1XwTm53GwEMr5MJCjHNVe2G8np/u4vpM83EiUJGjKUQx5xwh7UUccwcbTaexr3dCLpzmrexmBhFojMVr3ud4IwIwuXEav7PYoZd9XuN7Iu/fQHxe885vOfGOlSN7x7Ix9UZxeRFoxMIINRbsrdOvupfNTxOFvOghKT8dkd48eO/N06+eCLhZJwIRWA8LHgAQphP9ONJFoHFeXvNXcrJFNybU0zZVXZur2gvj9XzZXNBHmtdGBCpy4LQLUTjNh/YiLq/5FzKdxkXYL0RfnzLr662y5twbTr4WgcbYvOYNmDmNY/U5q+8ijrkrrBovAjDA3KyPtHb9a81nI6pq58BpvqC6F+05VkCosXC/PP3qyQ/Ng3d/CpEBCbi4F59/3zTvvXX6dBBwt04EIrA8FjwAII5O9ONIjxeBxiV4zf85bzRO1FL9Pz0bjYsdc303FojrhJz1mreqq3EaAxVUfEvPUkEoQ3sxD69lxlPWjxBQTtNG4zPcaV5fiu9/LksEYFoRUE5VqzHMOGeFVbNEhV8AOxvNPxfbaZ4KgAgzvWPNFVSnPcdqCDVW4OHpU28hsm0z68sJcLdt8+Wb3/7uXbsnBRyu0zgJzW5czM3uMRY8ACCuTvTjSIcXgcYleY2f96nm4zQukjKhnoZWuyqacyyOTlVWe6EESy2KOI1tBNVd02AVoZYIQk1hnV6Yi9f8/bxx2gWUHwlrazVvX7/PQhlsXpnfRvMHYAgop8Np/C7smrtNnaqsegFYUqf523UnKvKmZKPdO9Yc4zPac6yKUGMlLET2i9OvNmqolIJVvGiaBx+99e+/YxICx+rFgAnzYoEUAObTi34c6/PiPlyDjbE+0HIT6ktU/cLrWu0WR1vNw2sMKA1CSbyWC5tvtDsuzQlLazV+/nNXhDJe8x6rip2l+nnjNFaeIaC8jlbz9/UTNh0vr7u4PtO8nHbj9Y2wNOt7p81greZn/cJGbHAF1tJpHKPN/Qy2YgPZmlrNv9lkqppNe47VEGqszFunX3U/NA9+xXHUWEzTDHYE+punT3sBYShtjbl4seABAHOjH8eaBs1/zClu12n+hVLjtNyCO5YLOAziGS6Z1/j9fqlldBrvWcKNy2i1ayec5jeI9mINnZbp581GLJ4vqdWyYysvNh2vxULnS7SfToQblzQdR2qf9xJFRyz0Yv1BJwBrs2raS42LN2J8tqRWy4zPrD2nYBVWR6ixQuNx1P/3XTXbpSYLUatGn791+rv3OG4aEXmNg/C5j7ZBPQax4AEAS/GiH8fy7L3XFkbZUby+pRZKTatxctcqOm2E2FotF3CwKi88w3WwxZKlTphx2oUbWXibx0bLh8zp89e1ZD9vNtotnnMsdXytln+GBy1XvRfXW3IzohPhxjk57Sozdpq/SrLxGu+fJwKQCq/xuRy0jI0IN87pRMuMz6aT7WjPkQRCjZV6eNq/eOv03z7dNg8+omojYvvpnmoevGeVQQXEt+TRNigbC6QAsDz6cSzJ7jN2FKdlWihdarHaQg77i6VOCNFq2aMnrb/ohJp0WjYU5UQwKqapGtR3Gj/PVsuYjqqlz1/f0v282WjcxHAmglExtFqn6vUUSvbC2ryWraDstBuvfyHG66Fajc/vVJlxiTCjGUQoGUiV1/h8LjkXu9HYDjE+C7f/jvWN5h+fWTtu44BBQCIINVbuF6dP+++b5r1to68FxLBtvvy++fm7b50+HQTMq9O4yOUF3M+04NEJALCWTvTjmM+0o7gTUuQ1TpAuOaHuNC6W/kEEl+5rfwJ9qYDDICr71mwKRQ1a1kYEo47VavlqUJNp0a0XUuG1fD9vWu2CURwvfz9r9PWTaRMDoeS02Pdi38lSx8ob9+qfOW00aIVD2TNsn90agWQzFQ7wApCyTuO6mNdyWjE+O1ar5d+x2GSCJDUCXvnLycebB9vt4622TsA9NWqeb5vmM8KMWIHT+LLuBNzNFjwI0dyf0/jyVDrGxsvqNE5mlKzXOFmEmzmNu0wJGCEW+vq82NE5a1Vk8RoX4AZxv1xlk+X23Xyo5RdFbRI9lWBDq/Fd81i/EvdWqE7rjRe9xvbha1Gl4jrWTnyi8TlptQ5rLzpx+kLK1uznzaDxGe6Fq9bs6yeDlg9X4P6c1pt79xo3uXwp7pPrtNr1xUtuKJh4jc/wIMyh19hGH2uNTULIg9O6a6qDxvGZte+M4y9b6x3LvgebH3uiNGx1PK9xLgYFYeEWl3x3snn7Df1Tp23ziYDDvLhoSb7kqGkkoFP54RiEYcHjeE6EGhFfJ0KN2OlEP45w9PV5chq/t5AFm1CDdsElrzrZ5LkFzO17ONHyC6Ne6S2KtiLUmAKn9TcyehFwNNYubC6u97VuBS0vQhQ5cVq/n7exoS2cf6u6qwCv3dfvs4XzTsiJBR3WXDe0zWtTCMarXq3GfnijdZ9h+x6sL+bdez69CDViXp3WnYvdH58Nqrc9WfsdK8XN4YQacQkLt7jWdycfu3/abs+o2ohbNc3wg5qPHp4+9QLSsPYOcKRpOspmEI7lRKgR8XUi1IjLnKi+jONYX2/PWs2L1CWw6nzWL6y5OGcG7SbVn6ts+5Pnj7TeZ59qILkVocZU2L3Zad0wxcTrcjtR+uKbtQ3TAlur9RGiyFcq/bzZX0D3Klsqff3Ei1ByzjZK4/hQG6MP2j3HJZuqqv5G64eRTWrVvErWi1Aj5ueUzlxsTeOzVN6xUjopYx+hRlzCwi1uxZHUuE6jxm+b5iOOmkainNbfAY50DBoDjSx4hHEi1Ij4OhFqxPU6UbURhxvEsXUlcRfXM6URnDFe46Lpt69+5h5ynBZFf/3qp9O6vNIONrQi1JiaVmMb4ZSO4eL6vXZB6NzfPZ0utxMpBNAMGxjK4DRuRj5ROqb+ferrvfJmz2yrXQDKKR1UVS+DU1pz73Y/7T/Dg/I2VVSdgsit0jGId+8l9SLUiGWktIFs4rULr5cwPnNK6x3LK+25GEKNuISFW9zJqja+oZebi+aDhUX8dNT0D/rnJw9PnzD5gNRtlMbOUayDXaNxORFqRHydCDXiZk5UbcTt6OvLtlGaY/lp0fT32k2spxp0nBZE7bKJ81YEG+6rFaHGFKW46LZvClT8UbvFuFQ5jW2E/bQAVKt0Qoz7CEKVZ6N05+y8Lvf1KYeVU+/rJ8PF9ZnKr4Bdm43SHq9Pz7FX2uN1p/HZtWd4ep5Tw8aCdfQi1IhlOaU7F+uVz/jMKe13rGkuNeV3K0KNuISFWxzMwo0/08uu2VL9rEKEGZErJ6o21ojjqOJzItSI+DoRasTdOlG1Ea8bRIWIGjjlMZafFk7t5x/3fr3EJPvb2gUa3MX1ji5PoKdoUD7BhlaEGlPmlM8GiCkEbdefdLmdmJvTrp2wn1NoYvrzKfPimNqSOeUzZ7cfkrJn2O9d9PU3YxNS+eze/FR5vLNPffH+eN1r/rHa/jM89cNOeTzDho0F6+lFqBHr6JRHu77m+Mwpv3esQfnMxRBqxCUs3OLeCDfWpWma/ns1nz88feoF5GsjqjbWwIsFj7k4EWpEfJ0INeIwTmxSwMiLvr5GJxqPqnTKk9dusXT6+acr//t1pgVQ88u9P54quuz/7znIMdjQilBjDjbK+31/ClW8uHL9ee9/v2lRzu39+p1XP/fbCKc82X/vFKJA+Zzyr9Dudbmv33+GpXr6+n29xoVzglB1cErvaPn78Hq9Hz52vG6c8n+GB41j90FYSy9CjViPU/5zsV7Hjc/c3q9LeMfKcS6GUCMuYeEWRxuPpd4+03bbCsUhzIhCdaLaU4mmBY/US6bnzIlQI+LrRKgR95N7sAnHo6+H2YiNSrnKtcJLK0KNuXDatRHIWy+CULXaiH6+BIMIQtVsI57j3HmNz3AvrK0XoUasr724nol2PVc5HDV9HUKNuOSBgCNZ2O2t09+990Pz4FfbRl8LRbAwo32nb57+7iMCjShQp3EwQ5tVjv7ielccgwEANTjV2I9z7HBd7Hunr4fpNS7MfC7kYtDYbn8qnl/My4v3/dwNGtt4G+fRXtSp1/gMWz/vhdz4i+sDEaKpXS/e2XM1VfKyd+9eADAaRLueo0Hj99aJdysUgFAjglnw7RenX20IN2bthRp9/kPzzw8JM6ICXuOuURvQDUKuBu0WPLwAADXpxYRaDQaNfb0tjnoBIy+CSzkYND6/dnkBy/Have/TRuRh0K69GASM/fy0icELqbOFcnsvs3b3VMCo1xiO4zlO3xRmJPwC4Da9mIvNwSDmYlAgQo2I5mq4sVHjhdRNYcZfvXX6Vffw9AkvLKiJFwvlORrEggcAYNSLCbUSDaKvx928CC6laBDPL9LgRRuRukG0F7iZF+HG1HmNR8VbO9sLeJ2tNXXiOU4VYUYAx+jFXGyKBvFuhYI1Amb0l5OPN41efqKtHgnpaJrh4v99/tbp00EAJpuL6/HF5YQUDRonWgZhDe7iOlf5GBsvq9PY7pas1zjJg2VsRF+es0H09Tie09iv/Ea0AWsYVObz215cZzqeLfZ4IQVOYxvxobC2QfT3uL+3L64TMdZPwaAxLN4LuL+NeI7XZuHFLy+uJyLImLpeYWNXwk1Ywka062saVOa71VbH8xrnYlAQKjViVr84fdq/dfrVu02zfZfqjav7qSpj07zx7lunv3uPQCPwml7jQIeXvbQMYocRAOAwvXa7hQchF4Po6xHOa5xMt/uIigHL6cXzizx47So30kYsbwpQMOeCY9k91Iux/poG7fr8XsBxevEcr2XQ+Pw+FJUZAcTTi3Z9abxboTpUo8HixuqN2w+13bbC3F6oaZ5TlRE4ihOVHNZig3Lb9W07Rr2QAicqNSK+TlRqxLycxvvsfY3VXZAOqkNgCVbR6cNXPxFPTc9vKyo1low2Yn7DxfWtxjEx/T1ic6JK89wYs2NuTryzz2maYz8VoZcc9aJSI/LjxPhsLoPqebeiUiMuYeEWq/nu5GP3s4sJYgKOM2ia4aW2377UP/cPT58w4QCEcRoXsyihPr9BLHikyolQI+LrRKjx/2/vDnLiOKIAgP6GOHLkRbiB6wZwA09OEOcGbLKOcwPnBMkyuzg34AjtG9g3aE4QIiWRpYTpdDGUugeDPcAwTPe8J5VqAAtLQP1f/et3D5tR3q4uF4NnwWOqQ65n81LYz69DHbv3lrGz0NS4C1Isftc/dOMouC8NFDyG47DXX6c6vE08m+Wafb3q6HOx6+7xehOaGhm34xDX72tXr600NbLEwS1bQYPjvV08kbGN+e/n8exEIyM8mFn0G3HWw4HHOKTQ1Mj6vQ5NjWxeij6Xp2ATyhNe3oTmHh5fblh6FZ4asKo6drsReRaaGndNjhHHsXhiVApWleNDvqbP1/Z1wONJsYjdDtBvrw43H7EdUrjZ4C7qsIan5k1oamQaUojrt5Fj+LtY1FLr2M2YrqmRJQ5u2Tp/vHx1sB//zLqXL/fa6kUbbQo+0kXzpqqq7iKlOvkvnr7TyAgblUKR9D7Kpvyny1n82n6pG7/F9H0TbNJxTL9JPBeUfwm2VWlceBGKauvmpgXGYBaLJ8KIAcvqcCBa5L+Ln+Puvgs/wzHLv/9ZLBocZ8FVw1zvup5tlGJ5DR8EQ6U2J+ezzVKowd/EGp6+17G4Vr2rH2PxNwLbJIW4fp1yk9jb8JTd7D43lzbhIROTo6mRrffXy++PzqOa7eXNW7vTBYizqqpOzqN9/2XsnXx18msTwDZIYRO+iuGTGxx4ALBtUsjn91XHovhWh0ZGxifF7jY+KJ7D56VYxIY8dvVJr8PmiTzXAeMyCzcz5LVb8r3aHGOT9+ez6Ndxit3TxCIPW8PAFOx6XB/uy+oAbqSpkdHJTY7z2O8KD/NZ9wd8GO1EixBVTmbV2zbm757Efq2JEUZBcWVZ3Y33YVMOwLjkfJ6vMTzB7dM0QjFls1is/RcxvSbHsnbzPr0OT++Au0jRNzkexjT3ClebGDVPMCWlfpfHYUzzpqayhku+r8MaZlpSTD8Xl4aXPLvmBqYuxbTjeonpddiXwa1oamT08ttVfxEfjuYxP9qLKsVFo2ObE91YDh3OoqouCgy5gXGvS2r/xrPG20nDJBxdjm8v5xTTZlMOwBSVJsdZ9E2OU2pwWlUTyzcsNAG7I0UfB0pxfQxxoInFHv00+ierNQGsW9krlGboFOM6hGtCrICjwRhTrs9y/a2JRU0uz/XlrC7HLhlzLi5ruDQiu5kAYLxxXUyHNdPUyGSVZsc2zlMVVYqLYkR10LZt9/FmG4vaLnlVVdV0/2/TRnvajUbzIuykFOM8DL1OE31jg005ALum3Kwwi0VOL4W2qWhCnofPKeu+zCUWpNhszaGJ/mlMf0a/ZsvngcczjBGpG8+jjxGbrgU0g3Ea/UFbnsUKuN5w/ebxWLk+a+LjfF8+bgK4yXAN5zFcx5vMxaXJJY/TwWtrGOB2SlzPc7m+2nRd9iz6fZiYDg9MUyM7Kzc9Pom/Uxv7B23Mu2TXHlw2P17oXj9f5fvkJsU8z7vkVUXbJbCqm/fOvuiS1od4eqZpEfiMFP0GPM+H8TjF0esM7yhyQAoAn7YtDU6ragYjNy/m3F5HX5gD7md4WHow+Lh87esVv8/p5dxczmWPfhaK5TB2Kfq4cHW+S5wY5vAm+lghr8PD+FSuz1Y6X4hFzW24dstc1m8TwEO4um7T4HPl66vk4uvWsD07wOZdtzcbxvVV92ZXr6+GezLXV7BhmhoBYHul6DfgZX4ey5vwdOXf32S40b66ES+FlyaW7xq1MQeA9UmxfFiSRzkkuSm3Dz9/k2bwepUc71AFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAuIP/AeJJeVlU/8lsAAAAAElFTkSuQmCC"/>
        </defs>
    </svg>
```

Reply:
Yep, we can add the image to the `assets` folder. For simplicity sake, I would recommend just using the `.png` file.